askama_shared/src/generator.rs

```diff
@@ -1230,13 +1230,19 @@ impl<'a, S: std::hash::BuildHasher> Generator<'a, S> {
     fn visit_lit(&mut self, lws: &'a str, val: &'a str, rws: &'a str) {
         assert!(self.next_ws.is_none());
         if !lws.is_empty() {
```

Comment:
We should flip the condition and turn it into an early return.

Reply:
We can't make it return early, some code after this condition needs to be run.